src/components/task/taskDetail/TaskCommentDropDown.tsx

```
@@ -0,0 +1,121 @@
+
+  useEffect(() => {
+    const handleClick = (event: MouseEvent) => {
+      // 트리거 영역 클릭 시 토글
+      if (
+        triggerRef.current &&
+        triggerRef.current.contains(event.target as Node)
+      ) {
+        toggleDropdown();
+      } else if (
+        dropdownRef.current &&
+        !dropdownRef.current.contains(event.target as Node)
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)